### PR TITLE
Add Example command how to run api-functional test

### PR DESCRIPTION
**Pull request by nuzil**
## Purpose of this pull request

Add Example command how to run api-functional test

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/get-started/web-api-functional-testing.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/get-started/web-api-functional-testing.md [hunk 1]
```diff
@@ -106,3 +106,9 @@ interface AdapterInterface
 1. Configure your DB connection and install settings in `/dev/tests/api-functional/config/install-config-mysql.php`. Specify the Magento database. The base URL to access this Magento instance must be the same specified in the `phpunit_rest.xml` or `phpunit_soap.xml` file.
 
 1. Run `phpunit` using the `/dev/tests/api-functional/phpunit_rest.xml` or `/dev/tests/api-functional/phpunit_soap.xml` configuration file.
+
+Example:
```
Comment (dmytro-ch) [inline src/guides/v2.3/get-started/web-api-functional-testing.md:110]: Hello @nuzil.
Could you please remove the trailing space in order to fix the failed test?
Thank you!
```suggestion
Example:
```

Comment (nuzil) [reply src/guides/v2.3/get-started/web-api-functional-testing.md]: I was faster then you :-D

Comment (dmytro-ch) [reply src/guides/v2.3/get-started/web-api-functional-testing.md]: Great, thank you :)

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**